ingestion/src/metadata/orm_profiler/orm/types/bytea_to_string.py

```diff
@@ -49,4 +50,11 @@ def process_result_value(self, value: str, dialect) -> Optional[str]:
         if not value:
             return None
         self.validate(value)
+
```

Comment:
@MilanBariya could you please add a try/except block around the logic and fall back to default return if anything goes wrong while decoding